WalkingApp/dialogs.py

```diff
@@ -1,11 +1,17 @@
 import flet as ft
+import model
 
 
 def _close_dialog(page, dlg):
     dlg.open = False
     page.update()
 
 
+def _clear_table_and_statistics(page, dlg):
+    model.reset_database()
```

Comment:
This module should only deal with displaying dialogs, it should not access the database. 
The controller (`statistics_r.py`) should define what happens when the user confirms the deletion.
It can pass a function to `show_confirm_deleting_records_dialog` that will be executed when the user clicks "Yes".